WalletWasabi.Fluent/ViewModels/Login/PasswordFinder/SearchPasswordViewModel.cs

```diff
@@ -38,30 +38,23 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 	{
 		base.OnNavigatedTo(isInHistory, disposables);
 
-		var cts = new CancellationTokenSource()
-			.DisposeWith(disposables);
-
 		_model.Progress
 			.Do(t => SetStatus(t.Percentage, t.RemainingTime))
 			.Subscribe()
 			.DisposeWith(disposables);
 
-		var t = FindPasswordAsync(cts.Token);
+		var findPasswordTask = FindPasswordAsync();
 
-		Disposable.Create(
-				async () =>
-				{
-					cts.Cancel();
-					await t;
-				})
+		Disposable.Create(async () => await findPasswordTask)
```

Comment:
The task never gets canceled... which should happen when the dialog gets closed during the search.

@adamPetho @turbolay